client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -30,28 +30,37 @@ import { isPreloadingCache } from './state/cache/selectors';
  */
 const renderCachePreloadInterval = ( {
 	handleChange,
-	isRequesting,
-	isSaving,
+	isDisabled,
 	preload_interval,
 } ) => (
 	<FormTextInput
 		className="wp-super-cache__preload-interval"
-		disabled={ isRequesting || isSaving }
+		disabled={ isDisabled }
 		min="0"
 		name="preload_interval"
 		onChange={ handleChange( 'preload_interval' ) }
 		step="1"
 		type="number"
-		value={ preload_interval || '' } />
+		value={ preload_interval || '0' } />
 );
 
 class PreloadTab extends Component {
-	state = {
-		preloadRefresh: true,
+	componentWillReceiveProps( nextProps ) {
+		this.state = {
+			preloadRefresh: parseInt( nextProps.fields.preload_interval, 10 ) === 0 ? false : true
+		};
 	}
 
 	handlePreloadRefreshChange = () => {
-		this.setState( { preloadRefresh: ! this.state.preloadRefresh } );
+		this.setState( { preloadRefresh: ! this.state.preloadRefresh }, () => {
+			const { fields, setFieldValue } = this.props;
+
+			if ( this.state.preloadRefresh ) {
+				setFieldValue( 'preload_interval', get( fields, 'minimum_preload_interval', '0' ), true );
```

Comment:
Apparently `minimum_preload_interval` will be either 30 or 720, so I've [defaulted it to 30] (1b8cc72).

> And as above, can we use an integer instead of a string?

Yup, did that [here](https://github.com/Automattic/wp-calypso/pull/14518/commits/8d50eb696d70f48ddd677a3a2a274e0ee5b923f6), but you'll need to test using [this branch](https://github.com/Automattic/wp-super-cache/tree/fix/disable-preload-interval) of WPSC, or a 0 value won't save.